bin/host/src/handler.rs

```diff
@@ -111,47 +132,121 @@ pub async fn fetch_eigenda_hint(
         recency_be_bytes.to_vec(),
     )?;
 
-    // pre-populate validity address. Currently, assume everything is correct
-    // (ToDo bx), after proxy returns error code indicating cert is wrong, route
-    // with appropriate response false
-    let claimed_validity = true;
-    let mut validity_address = altda_commitment.digest_template();
+    Ok(())
+}
+
+/// Process response from eigenda network
+async fn process_eigenda_response(
+    providers: &<SingleChainHostWithEigenDA as OnlineHostBackendCfg>::Providers,
+    altda_commitment_bytes: &Bytes,
+) -> Result<(bool, bool, Vec<u8>)> {
```

Comment:
Think a better function name would help here. Also I'm very scared whenever I see 2 arguments/return-values that have the same type (here (bool, bool)), because its easy while refactoring to make a mistake. At the very very least if you don't want to introduce specific types for is_valid and is_recent, then document the return values.